### ISSUE TITLE
GHC 7.10.2 build error

### ISSUE THREAD

**Issue by peti** [alanz/haskell-token-utils#2]
haskell-token-utils-0.0.0.6 doesn't compile with GHC 7.10.2: http://hydra.cryp.to/build/1062754/log/raw.

**Comment by alanz** [comment]
It never will. I guess I need to change the cabal file to reflect that. For 7.10.2 it is replaced with ghc-exactprint

**Comment by tebello-thejane** [comment]
``HaRe`` relies on it. Should ``HaRe`` be updated to use something which works on GHC 7.10.2?

**Comment by obadz** [comment]
As I understand it, latest `HaRe` now depands on `ghc-exactprint` and not `haskell-token-utils`.

cc // @mpickering

**Comment by alanz** [comment]
Correct. So haskell-token-utils is a legacy package for use with HaRe
0.7.2.8, and will not be updated.
